corehq/apps/hqwebapp/static/hqwebapp/js/bulk_upload_file.js

```diff
@@ -7,17 +7,11 @@ hqDefine("hqwebapp/js/bulk_upload_file", [
 ) {
     $(function () {
         "use strict";
-        var ids = [
-            "#bulk_upload_form",
-            "#bulk_ui_translation_upload_form",
-            "#bulk_app_translation_upload_form",
-        ];
-        for (var i = 0; i < ids.length; i++) {
-            if ($(ids[i]).get(0)) {
-                $(ids[i]).koApplyBindings({
-                    file: ko.observable(null),
-                });
-            }
+        var $form = $("#bulk_upload_form");
+        if ($form.length) {
+            $form.koApplyBindings({
+                file: ko.observable(),
+            });
```

Comment:
just confirming that this is changed because you replaced the file [here](https://github.com/dimagi/commcare-hq/pull/23391/files#diff-a638aa527bc016b24dad9571d2a64ff7R19) with translations.js and all other places for this js would only have the first id?

Reply:
Yes, exactly. I didn't think ids specific to app manager belonged in this more general file, and this file does so little that it's easy enough to have app manager apply the bindings instead of having it include this file.